### PR TITLE
feat(index): add sourcemap support (`options.sourceMap`)

### LINKED ISSUE #30: SourceMaps
I'm requiring a js file from my project, and I have devtools enabled with webpack. When I go to Chrome Devtools and I try to check the file, I only can see a compiled file like this:

module.exports = "[all-the-code]".

So... it's impossible to me to debug using devtools.

### PR DESCRIPTION
**Pull request by jiaweihli**
### PR (Notable Changes)

```
chore: update `loader-utils` v0.0.0...1.1.0 (:label: Major)
feat: support sourcemap support (`options.sourceMap`)
```

Global options like debug are deprecated[0] in `webpack >= 2.0.0`, and require
`LoaderOptionsPlugin` to be enabled.  This commit adds support for a loader
`options.sourceMap`, to enable configuring this on a loader/require level.

[0] https://webpack.js.org/plugins/loader-options-plugin/

## Issues

Closes #30

> Edited by @michael-ciniawsky (Formatting && Grammar)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ Executes JS script once in global context.
 
 > :warning: Doesn't work in NodeJS
 
+**Options** (Webpack 2)
```
Comment (michael-ciniawsky) [inline README.md]: <h2 align="center">Options</h2>

|Name| Type|Default|Description|
|:-----:|:----:|:------:|:-----------|
|`**sourceMap**`|`{Boolean}`|`false`| Enable/Disable Sourcemaps|


```
<h2 align="center">Options</h2>

|Name|Type|Default|Description|
|:-----:|:----:|:------:|:-----------|
|**`sourceMap`**|`{Boolean}`|`false`| Enable/Disable Sourcemaps|
```

**Comment by michael-ciniawsky** [comment]
We have no tests in place here 😊 , if you have an idea to add a quick test setup for the loader aswell please go ahead as otherwise forget about it 🙃

**Comment by jiaweihli** [comment]
Bumped with your suggestion.  I don't have an idea for the test setup yet, maybe in a future PR!

File: package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "author": "Tobias Koppers @sokra",
   "description": "script loader module for webpack",
   "dependencies": {
+    "loader-utils": "~1.1.0",
```
Comment (bebraw) [inline package.json]: This is webpack 2 only, right?

Comment (jiaweihli) [inline package.json]: Not sure - it's supposed to work for webpack 2, but it doesn't specify the behavior for webpack 1: https://github.com/webpack/loader-utils

Comment (bebraw) [inline package.json]: Ah, ok.

**Comment by michael-ciniawsky** [comment]
@jiaweihli The changes look good. Please rebase against current `master` and fix the commit author for the CLA (or open an new PR). Otherwise I'm going to close this PR within the next ~48-72 hours due to the missing CLA signment and inactivity

**Comment by michael-ciniawsky** [comment]
https://github.com/webpack-contrib/script-loader/pull/31#issuecomment-420657577

Feel free to reopen if the comments are addressed and the CLA is signed. Thx

**Pull request closed without merging**